HousingRepairsOnlineApi/Gateways/AppointmentsGateway.cs

```diff
@@ -62,5 +63,27 @@ public async Task<IEnumerable<Appointment>> GetAvailableAppointments(string sorC
             await httpClient.SendAsync(request);
 
         }
+
+        public async Task<CancelAppointmentStatus> CancelAppointment(string bookingReference)
+        {
+             // returns appropriate status code - 200, 404 or 500
+             var request = new HttpRequestMessage(HttpMethod.Post,
+                 $"/Appointments/CancelAppointment?bookingReference={bookingReference}");
+             request.SetupJwtAuthentication(httpClient, authenticationIdentifier);
+
+             var response = await httpClient.SendAsync(request);
+
+             switch (response.StatusCode)
+             {
+                 case HttpStatusCode.OK:
+                     return CancelAppointmentStatus.Found;
+                 case HttpStatusCode.NotFound:
+                     return CancelAppointmentStatus.NotFound;
+                 case HttpStatusCode.InternalServerError:
+                     return CancelAppointmentStatus.Error;
```

Comment:
Nitpick: this line could be removed which would then be combined with the case below (`default`), reduce the code and potentially improving readability.
```suggestion
```